onevizion/trackor.py

```diff
@@ -484,26 +484,28 @@ def get_filename_from_cd(cd):
 
 
 
-	def UploadFile(self, trackorId, fieldName, fileName, newFileName=None):
+	def UploadFile(self, trackorId, fieldName, fileName, newFileName=None, byteString=None):
```

Comment:
Propose to make ```fileName``` optional, and use only ```newFileName``` as filename if ```byteString``` is present

Reply:
Done